doc/manuals/install/index.md

```diff
@@ -91,7 +91,7 @@ a new `cctools-dev` environment via Conda:
 
 ```sh
 unset PYTHONPATH
-conda create -y -n cctools-dev -c conda-forge --strict-channel-priority python=3 gcc_linux-64 gxx_linux-64 gdb m4 perl swig make zlib libopenssl-static openssl conda-pack
+conda create -y -n cctools-dev -c conda-forge --strict-channel-priority python=3 gcc_linux-64 gxx_linux-64 gdb m4 perl swig make zlib libopenssl-static openssl conda-pack packaging cloudpickle
```

Comment:
what needs `packaging`? I don't remember seeing that one before.

Reply:
it's in `poncho_create` [here.](https://github.com/cooperative-computing-lab/cctools/blob/1ee88e5bedd14690437e6d2d6e419e65ac105208/poncho/src/poncho/package_create.py#L20)